packages/server/graphql/mutations/updateTaskDueDate.ts

```diff
@@ -65,14 +65,7 @@ export default {
         publish(SubscriptionChannel.TASK, userId, 'UpdateTaskDueDatePayload', data, subOptions)
       })
     }
-    segmentIo.track({
-      userId: viewerId,
-      event: 'Task due date set',
-      properties: {
-        taskId,
-        teamId: task.teamId
-      }
-    })
+    analytics.taskDueDateSet(viewerId, taskId, task.teamId)
```

Comment:
+1 the argument order feels wrong, normally it's `userId` then `teamId` or `orgId` and then the element living in the team, like `meetingId`, so from my intuition I would expect `taskId` to be last as well.